create-git-tag.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
```

Comment:
Add `set -eo pipefail` to make sure the script will fail in case of an error.
And add an empty line to the end of the file ;)